### ISSUE TITLE
ValidatorFX depends on Linux JavaFX artifacts

### ISSUE THREAD

**Issue by jreznot** [effad/ValidatorFX#16]
See https://youtrack.jetbrains.com/issue/IDEA-269305

When ValidatorFX is added to pom.xml:
```
<dependency>
            <groupId>net.synedra</groupId>
            <artifactId>validatorfx</artifactId>
            <version>0.1.13</version>
</dependency>
```
For some reason, this artifacts contains the dependencies to the javafx-base, javafx-controls and javafx-graphics with the 'linux' classifier.

As a result, when I import this project on Windows, I get 2 copies of these dependencies: the first one is added by javafx-controls explicit dependency in the project pom, and downloads the windows-specific artifacts, the second one comes from validatorfx, and use 'linux'. This causes the errors in the module-info, like
Module 'com.example.demo1444d' reads package 'javafx.beans' from both 'javafx.base' and 'javafx.base'.

Please exclude javafx dependencies from the resulting POM, e.g. ControlsFX does not have dependencies on javafx libs.

**Comment by torakiki** [comment]
Could this be the cause? https://github.com/openjfx/javafx-gradle-plugin/issues/26

**Comment by effad** [comment]
> Could this be the cause? [openjfx/javafx-gradle-plugin#26](https://github.com/openjfx/javafx-gradle-plugin/issues/26)

Yes, this seems very likely. I hope, the gradle plugin gets fixed soon, then I will re-release ValidatorFX and the problem should be gone.

**Comment by effad** [comment]
The issue in the javafx-gradle-plugin is still not fixed. If I add 
```
	configuration = 'compileOnly'
```
to the build.gradle the resulting pom-File will no longer contain any reference to JavaFX at all, which may be an improvement but is not really a solution, because then `gradlew test` can no longer be executed from the command line.

**Comment by effad** [comment]
I've just released Version 0.1.14 which no longer contains any dependency to JavaFX at all in the POM file.
Of course one will still need some JavaFX version to use ValidatorFX but avoding a specific version and classifier your Problem with IDEA should be avoided.
